docs/src/authentication.md

```
@@ -0,0 +1,46 @@
+
+Most HTTP client libraries handle Basic Auth natively. For example, with curl's `-u` flag:
+
+```bash
+curl -u "abc123:mysecret" https://api.st0x.io/v1/tokens
+```
+
+## Error Responses
+
+| Status | Code | When |
+|--------|------|------|
+| 401 | `UNAUTHORIZED` | Missing or invalid credentials |
```

markdownlint-cli2 (0.21.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)